src/Security/MemberAuthenticator/LostPasswordHandler.php

```diff
@@ -183,7 +207,7 @@ public function forgotPassword($data)
         // Ensure password is given
         if (empty($data['Email'])) {
             $this->form->sessionMessage(
-                _t('SilverStripe\\Security\\Member.ENTEREMAIL', 'Please enter an email address to get a password reset link.'),
+                _t('Member.ENTEREMAIL', 'Please enter an email address to get a password reset link.'),
```

Comment:
Yep, my bad, I fixed a rebase the wrong way around. Will do for all `_t()` calls I can find :)